app/controllers/katello/concerns/api/v2/registration_controller_extensions.rb

```diff
@@ -18,16 +18,6 @@ def prepare_host
         end
       end
 
-      def host_setup_extension
-        if params['host']['lifecycle_environment_id']
-          new_lce = KTEnvironment.readable.find(params['host']['lifecycle_environment_id'])
-          @host.content_facet.lifecycle_environment = new_lce
-          @host.update_candlepin_associations
-        end
-
-        super
-      end
-
```

Comment:
We were going to move this to another issue, right?

Reply:
@jeremylenz : Yes..I filed this for the registration change: https://projects.theforeman.org/issues/35874 
Will update this PR.